sd

```diff
@@ -115,8 +115,18 @@ sd() {
   point_path="$sdd/$requested_point"
 
   if [[ ! -L "$point_path" ]]; then
-    echo "Can't shift to point '$requested_point' because it doesn't exist."
+
+    type fzf >/dev/null 2>&1 || {
+    echo  "Can't shift to point '$requested_point' because it doesn't exist."
+    echo  "Also fzf issn't in path so we can't fuzzy find a match, sorry."
```

Comment:
I'd leave the messaging as it was; there's no need to remind the user of the features they haven't opted into.